### DISCUSSION TITLE
SSML Tags

### DISCUSSION THREAD

**Discussion by emorissettegregoire** [deepgram/community#655] [General help] [answered]
Hi there,
I would like to know if it's possible to include SSML tags in a text string for the Text-to-Speech API.
For now, I haven't been able to make it work with something like this. The speech is correctly generated, but it does not take into account the SSML tags.

```
"<speak>This is an example of <emphasis level='strong'>strong</emphasis> emphasis. <break time='500ms'/> Now there's a 500 millisecond pause.</speak>"
```

Thanks a lot!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by lukeocodes** [comment, marked as answer]
hey @emorissettegregoire as of right now, we have no support for SSML. we are looking at it!

**Reply by emorissettegregoire** [reply]
Thanks for the quick reply @lukeocodes

**Reply by emorissettegregoire** [reply]
Is there a work around to handle pauses ?

**Reply by lukeocodes** [reply]
we are improving models all the time, especially around natural pauses. my suggestion in the interim would be to chunk your strings where you need pauses, and make individual requests for that audio. that will likely result in a small pause in playback, which you could increase yourself with code if necessary

**Reply by lukeocodes** [reply]
additionally, I have had some luck with introducing `-` and `–` where I need pauses.

**Reply by emorissettegregoire** [reply]
Gotcha, thanks again

**Comment by jkroll-deepgram** [comment]
Update on SSML in this thread: https://github.com/orgs/deepgram/discussions/1031
